src/GridView/GridView.js

```diff
@@ -64,17 +89,18 @@ class GridView extends ConnectionComponent {
 						currentEntity={entity}
 						getEntityType={this.getEntityType}
 						getChildEntityTypes={this.getChildEntityTypes}/>
-					<SearchBar/>
+					{filterBar}
```

Comment:
If you have no condition for `filterBar` you can put the function call here directly instead of using a variable.